app/controllers/topics_controller.rb

```diff
@@ -25,7 +25,9 @@ def create
   def show
     @topic = Topic.find(params[:id])
     authorize @topic
-    @posts = @topic.posts.includes(:user).includes(:comments).paginate(page: params[:page], per_page: 10)
+    @posts = @topic.posts.includes(:user)
+                   .includes(:comments)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.